video/mp_image.h

```diff
@@ -57,6 +57,8 @@ struct mp_image_params {
 
     enum mp_csp_light light;
     enum pl_chroma_location chroma_location;
+    // The image should be flipped vertically before rotating
+    int vflip;
```

Comment:
Any reason to not just use a bool here?

Reply:
btw. what about hflip? Maybe we should make it more complete while at it?

Reply:
no reason not to use a bool, mostly just habit of `int` as a bool. as for why not hflip, we also have `rotate` and a vflip and a rotation together can be an hflip. Since this is specifically "flip vertically before rotating" it creates ambiguity whereas hflip before rotation would require us to add 180 degrees to p->rotate.

Reply:
Also implementing a vflip in a driver is much easier for the most part, as we can negate linesize for some things (e.g. vo_gpu) but mirroring horizontally is more complicated.